qemu_v8.mk

```diff
@@ -297,6 +325,37 @@ $(ROOTFS_UGZ): u-boot buildroot | $(BINARIES_PATH)
 				-n "Root file system" \
 				-d $(ROOTFS_GZ) $(ROOTFS_UGZ)
 
+################################################################################
+# XEN
+################################################################################
+.PHONY: xen
+xen-defconfig:
+	$(MAKE) -C $(XEN_PATH)/xen XEN_TARGET_ARCH=arm64 defconfig
+
+xen-common: xen-defconfig
+	cd $(XEN_PATH)/xen && \
+	tools/kconfig/merge_config.sh -m .config $(ROOT)/build/kconfigs/xen.conf
+
+xen: xen-common $(BINARIES_PATH)
+	$(MAKE) -C $(XEN_PATH) dist-xen \
+	XEN_TARGET_ARCH=arm64 \
+	CONFIG_XEN_INSTALL_SUFFIX=.gz	\
+	CROSS_COMPILE="$(CCACHE)$(AARCH64_CROSS_COMPILE)"
+	ln -sf $(XEN_PATH)/xen $(BINARIES_PATH)
```

Comment:
That is causing some rebuild of the `.config` every time `make xen-common` is run and seems overly complicated.
How about:
```make
$(XEN_PATH)/xen/.config:
        $(MAKE) -C $(XEN_PATH)/xen XEN_TARGET_ARCH=arm64 defconfig
        cd $(XEN_PATH)/xen && \
        tools/kconfig/merge_config.sh -m .config $(ROOT)/build/kconfigs/xen.conf

xen: $(XEN_PATH)/xen/.config | $(BINARIES_PATH)
        $(MAKE) -C $(XEN_PATH) dist-xen \
        XEN_TARGET_ARCH=arm64 \
        CONFIG_XEN_INSTALL_SUFFIX=.gz   \
        CROSS_COMPILE="$(CCACHE)$(AARCH64_CROSS_COMPILE)"
        ln -sf $(XEN_PATH)/xen $(BINARIES_PATH)
```

Reply:
Will change